### ISSUE TITLE
Accessibility for Hero: Ensure keyboard focus is indicated visually

### ISSUE THREAD

**Issue by cjwest** [SU-SWS/decanter#402]
**Accessibility for Hero**
***Ensure keyboard focus is indicated visually (Severity: 7)****
 The link "Button label", with style .su-button does not have a visible indication of focus in high contrast mode.  
[User Impact] 
Users with low vision who enable high contrast mode will not be able to track their location on the page and tell when this link has focus.  
[Code Reference] 
`<a href="" class="su-link su-button">Button Label</a>`

**To Reproduce**
1. http://decanter.stanford.edu/section-components.html#kssref-components-lockup

**Expected behavior**

Provide a highly visible indication of focus that persists in high contrast mode. This can be done with CSS outline styles.

**Screenshots**
![image](https://user-images.githubusercontent.com/284440/56447949-29709e00-62c0-11e9-9add-58f07da4270b.png)

**Additional context**
https://stanford.levelaccess.net/public/reporting/view_module.php?module_id=9631559

**Comment by yvonnetangsu** [comment]
Outline has been added back to all Decanter buttons.
